py3status/parse_config.py

```diff
@@ -292,6 +310,41 @@ def make_value(self, value):
             return None
         return value
 
+    def make_value_from_env(self, value):
+        # Extract the environment variable and type
+        match = re.match('env\(([0-9A-Z_]+)(, *[a-z]+)?\)', value)
+        env_var, env_type = match.groups()
+        if not env_type:
+            env_type = 'auto'
+        else:
+            # Remove comma and whitespace from match
+            env_type = env_type[1:].lstrip()
+
+        conversion_options = {
+            'str': str,
+            'int': int,
+            'float': float,
+
+            # Treat booleans specially
+            'bool': (lambda val: val.lower() in ('true', '1')),
+
+            # Auto-guess the type
+            'auto': self.make_value,
+        }
+        if env_type not in conversion_options:
+            self.error('Invalid conversion')
+
+        # Check the environment variable
+        # TODO: dynamic support, (lambda: os.getenv(env_var))
+        value = os.getenv(env_var)
+        if value is None:
+            self.error('Environment variable undefined')
```

Comment:
please add the name of the env var in the error message

Reply:
Fixed!